purchase_order_report_selection/models/purchase_order.py

```diff
@@ -0,0 +1,20 @@
+from odoo import fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    report_field_ids = fields.One2many("purchase.report.field", "purchase_order_id")
+
+
+class PurchaseReportField(models.Model):
+    _name = "purchase.report.field"
+    _description = "Purchase report field"
+
+    field = fields.Many2one(
```

Comment:
please, use prefix "_id" for relation fields: 

```field_id```

Reply:
okey)